lib/auth/sessions_test.go

```diff
@@ -128,7 +128,9 @@ func TestServer_CreateWebSessionFromReq_deviceWebToken(t *testing.T) {
 		Id:    fakeWebToken.Id,
 		Token: fakeWebToken.Token,
 	}
+	var receivedWebToken *devicepb.DeviceWebToken
 	authServer.SetCreateDeviceWebTokenFunc(func(ctx context.Context, dwt *devicepb.DeviceWebToken) (*devicepb.DeviceWebToken, error) {
+		receivedWebToken = dwt
```

Comment:
How bad is this? `receivedWebToken` is shared between subtests but for now there is only a single test.